Need virtwho.properties.sample in Robottelo repo

For collecting virtwho-tests from `./tests/foreman/virtwho/*`, a `virtwho.properties` file is required, so there should be a sample file that exists in the robottelo repo.

Error message:
```
_________________________________________ ERROR collecting tests/foreman/virtwho/ui/test_*.py_________________________________________
tests/foreman/virtwho/ui/test_*.py:20: in <module>
    from robottelo.virtwho_utils import deploy_configure_by_command
robottelo/virtwho_utils.py:18: in <module>
    virtwho.configure()
robottelo/config/virtwho.py:351: in configure
    raise ImproperlyConfigured(f'Not able to find settings file at {settings_path}')
E   robottelo.config.base.ImproperlyConfigured: Not able to find settings file at /home/gtalreja/sat_workspace/robottelo/virtwho.properties
```

--- a/virtwho.properties.sample
+++ b/virtwho.properties.sample
@@ -1,0 +1,29 @@
+# Make a copy of this file named virtwho.properties
+# The section name should be the hypervisor type, one of the values: esx, xen, hyperv, rhevm, libvirt, kubevirt
+# Take the esx type as an example
+# [esx]
+# The hypervisor type, the same as the section name
+# hypervisor_type=esx
+# The hostname or ip or other values for hypervisor server, depends on the type
+# hypervisor_server=
+# Admin username to access hypervisor server
+# hypervisor_username=
+# Admin password to access hypervisor server
+# hypervisor_password=
+# The config file to connect hypervisor, only for kubevirt
+# hypervisor_config_file=
+# The guest ip or hostname in the hypervisor
+# guest=
+# The guest port for ssh connection, by default guest_port=22
+# guest_port=22
+# ssh username to access guest
+# guest_username=
+# ssh password to access guest
+# guest_password=
+
+# The SKU ID for virt-who plugin testing
+# [sku]
+# The SKU ID for physical vdc
+# vdc_physical=
+# The SKU ID for virtual vdc
+# vdc_virtual=

